tardis/plasma/standard_plasmas.py

```diff
@@ -121,7 +121,7 @@ def assemble_plasma(config, simulation_state, atom_data=None):
     kwargs = dict(
         dilute_planckian_radiation_field=dilute_planckian_radiation_field,
         abundance=simulation_state.abundance,
-        number_density=simulation_state.elemental_number_density,
+        number_density=simulation_state.calculate_elemental_number_density(),
```

Comment:
Missing input argument here for elemental masses

Reply:
should bre resolved with this